src/renderer/viz/expressions/basic/variable.js

```diff
@@ -66,12 +66,16 @@ export default function variable(name) {
             return true;
         },
         get: (obj, prop) => {
-            if (prop == '_resolveAliases') {
+            if (prop == 'parent') {
+                return obj[prop];
+            } else if (prop == '_resolveAliases') {
                 return resolve;
             } else if (prop == '_getDependencies') {
                 return _getDependencies;
             } else if (prop == 'notify') {
                 return obj[prop];
+            } else if (prop == 'blendTo') {
```

Comment:
Question: should we not add the `blendTo` check also in the `set` method?

Reply:
Unless we want to override the method manually by assigning to it, and I don't know why we would want to do that, no, it is not needed